scripts/build

```diff
@@ -15,6 +15,9 @@ if [ "$CROSS" = "true" ] && [ "$ARCH" = "amd64" ]; then
     GOOS=darwin go build -ldflags "$LINKFLAGS" -o bin/support-bundle-kit-darwin
     GOOS=windows go build -ldflags "$LINKFLAGS" -o bin/support-bundle-kit-windows
 fi
+mkdir -p dist/artifacts
```

Comment:
Why moved to `build` script?

Reply:
No good reason. Moved it back and rebased. Thanks.